mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/liferay-portlet.xml

```diff
@@ -5,6 +5,8 @@
 	<portlet>
 		<portlet-name>todo</portlet-name>
 		<icon>/icon.png</icon>
+		<requires-namespaced-parameters>false</requires-namespaced-parameters>
+		<ajaxable>true</ajaxable>
```

Comment:
We don't need ```<ajaxable>true</ajaxable>```, this is true by default. 

We don't want ```<requires-namespaced-parameters>false</requires-namespaced-parameters>```, it is a good pattern to namespace parameters and ensure separation of ajax handling in a portlet. Without this setting on false, weird things may happen if you have two instances of the same portlet on the same page.